Adios2 Build error

Hi all,

Trying build the [hpc3 at UCI](https://warpx.readthedocs.io/en/latest/install/hpc/hpc3.html) install. 

First, there's an error in the `install_gpu_dependencies.sh` file lines 18-29:

```
if [ -z ${proj-} ]; then echo "WARNING: The 'proj' variable is not yet set in your hpc3_gpu_warpx.profile file! Please edit its line 2 to continue!"; exit 1; fi


# Check $proj variable is correct and has a corresponding project directory ####
#
#if [ ! -d "/dfsX/${proj}/" ]
#then
#    echo "WARNING: The directory /dfsX/${proj}/ does not exist!"
#    echo "Is the \$proj environment variable of value \"$proj\" correctly set? "
#    echo "Please edit line 2 of your hpc3_gpu_warpx.profile file to continue!"
##    exit
fi
```

There's two `fi` statements with only one `if`. Deleting or commenting one out fixes the issue. 

Second thing: 
Following the hpc3 instructions and getting to the `install_gpu_dependencies.sh` installations gives the following error which has do to with adios2. I'm not sure what's going on but it seems like it's not detecting a couple of the libraries? I tried changing to the latest version of adios2-v2.9.1 but same problem. 
Here's the output from running: `bash /data/homezvol0/ernestob/src/warpx/Tools/machines/hpc3-uci/install_gpu_dependencies.sh` :



```
[ernestob@login-i15:~] $ bash /data/homezvol0/ernestob/src/warpx/Tools/machines/hpc3-uci/install_gpu_dependencies.sh
Cloning into '/data/homezvol0/ernestob/src/c-blosc'...
remote: Enumerating objects: 9019, done.
remote: Counting objects: 100% (1272/1272), done.
remote: Compressing objects: 100% (592/592), done.
remote: Total 9019 (delta 697), reused 1208 (delta 668), pack-reused 7747
Receiving objects: 100% (9019/9019), 7.87 MiB | 20.30 MiB/s, done.
Resolving deltas: 100% (5914/5914), done.
Note: switching to '7e4ff2d2bdb0db53fdf0d66d72aac5d0b5cc3a7c'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

-- The C compiler identification is GNU 11.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /opt/apps/gcc/11.2.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
Configuring for Blosc version: 1.21.1
-- Using LZ4 internal sources.
-- Using zlib internal sources.
-- Using ZSTD internal sources.
-- No build type specified. Defaulting to 'Release'.
-- Building for system processor x86_64
-- Adding run-time support for SSE2
-- Adding run-time support for AVX2
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE
-- Configuring done
-- Generating done
-- Build files have been written to: /data/homezvol0/ernestob/src/c-blosc-pm-gpu-build
[  0%] Building C object blosc/CMakeFiles/blosc_shared.dir/blosc.c.o
[  8%] Building C object blosc/CMakeFiles/blosc_shared.dir/fastcopy.c.o
[  8%] Building C object blosc/CMakeFiles/blosc_static.dir/fastcopy.c.o
[  8%] Building C object blosc/CMakeFiles/blosc_shared.dir/blosclz.c.o
[  8%] Building C object blosc/CMakeFiles/blosc_shared.dir/shuffle-generic.c.o
[  8%] Building C object blosc/CMakeFiles/blosc_static.dir/shuffle-generic.c.o
[  8%] Building C object blosc/CMakeFiles/blosc_static.dir/blosc.c.o
[  8%] Building C object blosc/CMakeFiles/blosc_static.dir/blosclz.c.o
[ 14%] Building C object blosc/CMakeFiles/blosc_shared.dir/shuffle-sse2.c.o
[ 14%] Building C object blosc/CMakeFiles/blosc_shared.dir/bitshuffle-sse2.c.o
[ 14%] Building C object blosc/CMakeFiles/blosc_static.dir/shuffle-sse2.c.o
[ 14%] Building C object blosc/CMakeFiles/blosc_static.dir/bitshuffle-generic.c.o
[ 14%] Building C object blosc/CMakeFiles/blosc_shared.dir/bitshuffle-generic.c.o
[ 14%] Building C object blosc/CMakeFiles/blosc_static.dir/bitshuffle-sse2.c.o
[ 14%] Building C object blosc/CMakeFiles/blosc_static.dir/shuffle-avx2.c.o
[ 14%] Building C object blosc/CMakeFiles/blosc_shared.dir/shuffle-avx2.c.o
[ 16%] Building C object blosc/CMakeFiles/blosc_shared.dir/bitshuffle-avx2.c.o
[ 16%] Building C object blosc/CMakeFiles/blosc_static.dir/bitshuffle-avx2.c.o
[ 16%] Building C object blosc/CMakeFiles/blosc_shared.dir/shuffle.c.o
[ 17%] Building C object blosc/CMakeFiles/blosc_static.dir/shuffle.c.o
[ 18%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/lz4-1.9.3/lz4.c.o
[ 19%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/lz4-1.9.3/lz4.c.o
[ 20%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/lz4-1.9.3/lz4hc.c.o
[ 22%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/adler32.c.o
[ 22%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/compress.c.o
[ 22%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/lz4-1.9.3/lz4hc.c.o
[ 23%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/crc32.c.o
[ 24%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/adler32.c.o
[ 25%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/compress.c.o
[ 26%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/deflate.c.o
[ 27%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/gzclose.c.o
[ 28%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/crc32.c.o
[ 29%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/deflate.c.o
[ 30%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/gzlib.c.o
[ 31%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/gzclose.c.o
[ 32%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/gzlib.c.o
[ 33%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/gzread.c.o
[ 34%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/gzwrite.c.o
[ 34%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/gzread.c.o
[ 35%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/infback.c.o
[ 36%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/gzwrite.c.o
[ 36%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/inffast.c.o
[ 37%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/inflate.c.o
[ 38%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/infback.c.o
[ 39%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/inffast.c.o
[ 40%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/inflate.c.o
[ 41%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/inftrees.c.o
[ 42%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/inftrees.c.o
[ 43%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/trees.c.o
[ 44%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/trees.c.o
[ 46%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/uncompr.c.o
[ 46%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/uncompr.c.o
[ 47%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zlib-1.2.11/zutil.c.o
[ 49%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zlib-1.2.11/zutil.c.o
[ 49%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/common/debug.c.o
[ 50%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/common/debug.c.o
[ 50%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/common/entropy_common.c.o
[ 51%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/common/entropy_common.c.o
[ 53%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/common/error_private.c.o
[ 53%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/common/error_private.c.o
[ 54%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/common/fse_decompress.c.o
[ 54%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/common/pool.c.o
[ 55%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/common/fse_decompress.c.o
[ 56%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/common/threading.c.o
[ 57%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/common/pool.c.o
[ 58%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/common/threading.c.o
[ 59%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/common/xxhash.c.o
[ 60%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/common/zstd_common.c.o
[ 62%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/common/xxhash.c.o
[ 62%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/fse_compress.c.o
[ 64%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/common/zstd_common.c.o
[ 64%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/hist.c.o
[ 65%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/huf_compress.c.o
[ 67%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_compress.c.o
[ 67%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/fse_compress.c.o
[ 68%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/hist.c.o
[ 69%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_compress_literals.c.o
[ 69%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/huf_compress.c.o
[ 69%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_compress_sequences.c.o
[ 70%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_compress.c.o
[ 71%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_compress_literals.c.o
[ 72%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_compress_superblock.c.o
[ 73%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_double_fast.c.o
[ 74%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_compress_sequences.c.o
[ 75%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_compress_superblock.c.o
[ 76%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_fast.c.o
[ 77%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_lazy.c.o
[ 78%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_ldm.c.o
[ 79%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_double_fast.c.o
[ 80%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_fast.c.o
[ 81%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_opt.c.o
[ 82%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/compress/zstdmt_compress.c.o
[ 83%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_lazy.c.o
[ 84%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_ldm.c.o
[ 85%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/decompress/huf_decompress.c.o
[ 85%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstd_opt.c.o
[ 85%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/decompress/zstd_ddict.c.o
[ 86%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/compress/zstdmt_compress.c.o
[ 87%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/decompress/zstd_decompress.c.o
[ 88%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/decompress/huf_decompress.c.o
[ 89%] Building C object blosc/CMakeFiles/blosc_static.dir/__/internal-complibs/zstd-1.5.0/decompress/zstd_decompress_block.c.o
[ 90%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/decompress/zstd_ddict.c.o
[ 91%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/decompress/zstd_decompress.c.o
[ 92%] Building C object blosc/CMakeFiles/blosc_shared.dir/__/internal-complibs/zstd-1.5.0/decompress/zstd_decompress_block.c.o
[ 93%] Linking C static library libblosc.a
[ 93%] Built target blosc_static
[ 96%] Building C object tests/fuzz/CMakeFiles/fuzz_decompress.dir/fuzz_decompress.c.o
[ 97%] Building C object tests/fuzz/CMakeFiles/fuzz_compress.dir/fuzz_compress.c.o
[ 97%] Building C object tests/fuzz/CMakeFiles/fuzz_decompress.dir/standalone.c.o
[ 97%] Building C object tests/fuzz/CMakeFiles/fuzz_compress.dir/standalone.c.o
[ 98%] Linking C executable decompress_fuzzer
[ 99%] Linking C executable compress_fuzzer
[ 99%] Built target fuzz_decompress
[ 99%] Built target fuzz_compress
[100%] Linking C shared library libblosc.so
[100%] Built target blosc_shared
Install the project...
-- Install configuration: "Release"
-- Installing: /data/homezvol0/ernestob/sw/hpc3/gpu/c-blosc-1.21.1/lib/pkgconfig/blosc.pc
-- Installing: /data/homezvol0/ernestob/sw/hpc3/gpu/c-blosc-1.21.1/include/blosc.h
-- Installing: /data/homezvol0/ernestob/sw/hpc3/gpu/c-blosc-1.21.1/include/blosc-export.h
-- Installing: /data/homezvol0/ernestob/sw/hpc3/gpu/c-blosc-1.21.1/lib/libblosc.so.1.21.1
-- Installing: /data/homezvol0/ernestob/sw/hpc3/gpu/c-blosc-1.21.1/lib/libblosc.so.1
-- Installing: /data/homezvol0/ernestob/sw/hpc3/gpu/c-blosc-1.21.1/lib/libblosc.so
-- Installing: /data/homezvol0/ernestob/sw/hpc3/gpu/c-blosc-1.21.1/lib/libblosc.a
Cloning into '/data/homezvol0/ernestob/src/adios2'...
remote: Enumerating objects: 88044, done.
remote: Counting objects: 100% (7308/7308), done.
remote: Compressing objects: 100% (1192/1192), done.
remote: Total 88044 (delta 6291), reused 6800 (delta 6103), pack-reused 80736
Receiving objects: 100% (88044/88044), 29.38 MiB | 16.18 MiB/s, done.
Resolving deltas: 100% (63420/63420), done.
Note: switching to '473fe8c7d1a13c0746910361aa45ee1b96f57bfb'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

Updating files: 100% (2698/2698), done.
-- The C compiler identification is GNU 11.2.0
-- The CXX compiler identification is GNU 11.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /opt/apps/gcc/11.2.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/apps/gcc/11.2.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE
-- Found Blosc: /data/homezvol0/ernestob/sw/hpc3/gpu/c-blosc-1.21.1/lib/libblosc.so (found suitable version "1.21.1", minimum required is "1.7")
-- Found BZip2: /usr/lib64/libbz2.so (found version "1.0.6")
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
-- Could NOT find ZFP (missing: ZFP_DIR)
-- Could NOT find SZ (missing: SZ_LIBRARY SZ_INCLUDE_DIR)
-- Could NOT find MGARD (missing: MGARD_LIBRARY MGARD_INCLUDE_DIR)
-- Found ZLIB: /usr/lib64/libz.so (found version "1.2.11")
-- Found PNG: /usr/lib64/libpng.so (found suitable version "1.6.34", minimum required is "1.6.0")
-- Looking for a CUDA compiler
-- Looking for a CUDA compiler - /opt/apps/cuda/11.7.1/bin/nvcc
-- Looking for a CUDA host compiler - /opt/apps/gcc/11.2.0/bin/g++
-- The CUDA compiler identification is NVIDIA 11.7.99
-- Detecting CUDA compiler ABI info
-- Detecting CUDA compiler ABI info - done
-- Check for working CUDA compiler: /opt/apps/cuda/11.7.1/bin/nvcc - skipped
-- Detecting CUDA compile features
-- Detecting CUDA compile features - done
-- Found MPI_C: /opt/apps/openmpi/4.1.2/gcc/11.2.0/lib/libmpi.so (found version "3.1")
-- Found MPI_CXX: /opt/apps/openmpi/4.1.2/gcc/11.2.0/lib/libmpi.so (found version "3.1")
-- Found MPI: TRUE (found version "3.1") found components: C CXX
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.4.2")
-- Checking for module 'dspaces'
--   Package 'dspaces', required by 'virtual:world', not found
-- Could NOT find DataSpaces (missing: DATASPACES_VERSION DATASPACES_INCLUDE_DIR DATASPACES_LIBRARIES) (Required is at least version "2.1.1")
-- Found HDF5: hdf5-shared (found version "1.13.1") found components: C
-- Could NOT find IME (missing: IME_LIBRARY IME_INCLUDE_DIR)
-- Found Python: /opt/apps/python/3.10.2/bin/python3.10 (found version "3.10.2") found components: Interpreter
-- Checking for module 'libfabric'
--   Package 'libfabric', required by 'virtual:world', not found
-- Could NOT find LIBFABRIC (missing: LIBFABRIC_LIBRARIES) (Required is at least version "1.6")
-- Could NOT find DAOS (missing: DAOS_LIBRARY DFS_LIBRARY DAOS_INCLUDE_DIR)
-- Looking for shmget
-- Looking for shmget - found
-- Could NOT find Sodium (missing: sodium_LIBRARY_RELEASE sodium_LIBRARY_DEBUG sodium_INCLUDE_DIR)
-- Checking for O_DIRECT
-- Performing Test O_DIRECT_WORKS
-- Performing Test O_DIRECT_WORKS - Success

-- ADIOS2 ThirdParty: Configuring KWSys
-- Checking whether wstring is available
-- Checking whether wstring is available - yes
-- Checking whether C compiler has ptrdiff_t in stddef.h
-- Checking whether C compiler has ptrdiff_t in stddef.h - yes
-- Checking whether C compiler has ssize_t in unistd.h
-- Checking whether C compiler has ssize_t in unistd.h - yes
-- Checking whether CXX compiler has setenv
-- Checking whether CXX compiler has setenv - yes
-- Checking whether CXX compiler has unsetenv
-- Checking whether CXX compiler has unsetenv - yes
-- Checking whether CXX compiler has environ in stdlib.h
-- Checking whether CXX compiler has environ in stdlib.h - no
-- Checking whether CXX compiler has utimes
-- Checking whether CXX compiler has utimes - yes
-- Checking whether CXX compiler has utimensat
-- Checking whether CXX compiler has utimensat - yes
-- Checking whether CXX compiler struct stat has st_mtim member
-- Checking whether CXX compiler struct stat has st_mtim member - yes
-- Checking whether CXX compiler struct stat has st_mtimespec member
-- Checking whether CXX compiler struct stat has st_mtimespec member - no
-- Checking whether <ext/stdio_filebuf.h> is available
-- Checking whether <ext/stdio_filebuf.h> is available - yes

-- ADIOS2 ThirdParty: Configuring perfstubs

-- ADIOS2 ThirdParty: Configuring GTest

-- ADIOS2 ThirdParty: Configuring pugixml

-- ADIOS2 ThirdParty: Configuring nlohmann_json

-- ADIOS2 ThirdParty: Configuring yaml-cpp

-- ADIOS2 ThirdParty: Configuring atl
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of double
-- Check size of double - done
-- Check size of float
-- Check size of float - done
-- Check size of int
-- Check size of int - done
-- Check size of short
-- Check size of short - done
-- Looking for include file malloc.h
-- Looking for include file malloc.h - found
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for include file stdlib.h
-- Looking for include file stdlib.h - found
-- Looking for include file string.h
-- Looking for include file string.h - found
-- Looking for include file sys/time.h
-- Looking for include file sys/time.h - found
-- Looking for include file windows.h
-- Looking for include file windows.h - not found
-- Looking for fork
-- Looking for fork - found
-- Found atl: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/atl/atl/atl-config.cmake (found version "2.2.1")

-- ADIOS2 ThirdParty: Configuring dill
-- Check size of void*
-- Check size of void* - done
-- Check size of long
-- Check size of long - done
-- Checking for module 'libffi'
--   Found libffi, version 3.1
-- Found LibFFI: -L/usr/lib64;-lffi
-- Enabling emulation
-- Found Perl: /usr/bin/perl (found version "5.26.3")
-- Looking for include file stdarg.h
-- Looking for include file stdarg.h - found
-- Looking for include file memory.h
-- Looking for include file memory.h - found
-- Looking for include file sys/mman.h
-- Looking for include file sys/mman.h - found
-- Looking for __clear_cache
-- Looking for __clear_cache - not found
-- Clear cache defined is
-- Found dill: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/dill/dill/dill-config.cmake (found version "2.4.1")

-- ADIOS2 ThirdParty: Configuring ffs
-- Check size of off_t
-- Check size of off_t - done
-- Check size of long double
-- Check size of long double - done
-- Check size of long long
-- Check size of long long - done
-- Check size of size_t
-- Check size of size_t - done
-- Looking for socket
-- Looking for socket - found
-- Found BISON: /usr/bin/bison (found version "3.0.4")
-- Found FLEX: /usr/bin/flex (found version "2.6.1")
-- Found atl: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/atl/atl/atl-config.cmake (found suitable version "2.2.1", minimum required is "2.2.1")
-- Looking for netdb.h
-- Looking for netdb.h - found
-- Looking for sockLib.h
-- Looking for sockLib.h - not found
-- Looking for sys/select.h
-- Looking for sys/select.h - found
-- Looking for sys/socket.h
-- Looking for sys/socket.h - found
-- Looking for sys/times.h
-- Looking for sys/times.h - found
-- Looking for sys/uio.h
-- Looking for sys/uio.h - found
-- Looking for sys/un.h
-- Looking for sys/un.h - found
-- Looking for winsock.h
-- Looking for winsock.h - not found
-- Looking for strtof
-- Looking for strtof - found
-- Looking for strtod
-- Looking for strtod - found
-- Looking for strtold
-- Looking for strtold - found
-- Looking for getdomainname
-- Looking for getdomainname - found
-- Check size of struct iovec
-- Check size of struct iovec - done
-- Performing Test HAS_IOV_BASE_IOVEC
-- Performing Test HAS_IOV_BASE_IOVEC - Success
-- Found atl: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/atl/atl/atl-config.cmake (found version "2.2.1")
-- Found ffs: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/ffs/ffs/ffs-config.cmake (found version "1.6.0")

-- ADIOS2 ThirdParty: Configuring enet
-- Looking for getaddrinfo
-- Looking for getaddrinfo - found
-- Looking for getnameinfo
-- Looking for getnameinfo - found
-- Looking for gethostbyaddr_r
-- Looking for gethostbyaddr_r - found
-- Looking for gethostbyname_r
-- Looking for gethostbyname_r - found
-- Looking for poll
-- Looking for poll - found
-- Looking for fcntl
-- Looking for fcntl - found
-- Looking for inet_pton
-- Looking for inet_pton - found
-- Looking for inet_ntop
-- Looking for inet_ntop - found
-- Performing Test HAS_MSGHDR_FLAGS
-- Performing Test HAS_MSGHDR_FLAGS - Success
-- Performing Test HAS_SOCKLEN_T
-- Performing Test HAS_SOCKLEN_T - Success
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
-- Found enet: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/enet/enet/enet-config.cmake (found version "1.3.14")

-- ADIOS2 ThirdParty: Configuring EVPath
-- Performing Test HAVE_MATH
-- Performing Test HAVE_MATH - Failed
-- Performing Test HAVE_LIBM_MATH
-- Performing Test HAVE_LIBM_MATH - Success
-- Found atl: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/atl/atl/atl-config.cmake (found suitable version "2.2.1", minimum required is "2.2.1")
-- Found atl: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/atl/atl/atl-config.cmake (found version "2.2.1")
-- Found ffs: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/ffs/ffs/ffs-config.cmake (found suitable version "1.6.0", minimum required is "1.5.1")
-- Found dill: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/dill/dill/dill-config.cmake (found suitable version "2.4.1", minimum required is "2.3.1")
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for sys/epoll.h
-- Looking for sys/epoll.h - found
-- Looking for hostlib.h
-- Looking for hostlib.h - not found
-- Looking for sys/sockio.h
-- Looking for sys/sockio.h - not found
-- Performing Test HAVE_FDS_BITS
-- Performing Test HAVE_FDS_BITS - Failed
-- Looking for writev
-- Looking for writev - found
-- Looking for uname
-- Looking for uname - found
-- Looking for getloadavg
-- Looking for getloadavg - found
-- Looking for gettimeofday
-- Looking for gettimeofday - found
-- Looking for getifaddrs
-- Looking for getifaddrs - found
-- Found atl: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/atl/atl/atl-config.cmake (found suitable version "2.2.1", minimum required is "2.2.1")
-- Found atl: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/atl/atl/atl-config.cmake (found version "2.2.1")
-- Found ffs: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/ffs/ffs/ffs-config.cmake (found suitable version "1.6.0", minimum required is "1.6.0")
-- Found EVPath: /data/homezvol0/ernestob/src/adios2-pm-gpu-build/thirdparty/EVPath/EVPath/EVPathConfigCommon.cmake (found version "4.5.0")

-- Found MPI: TRUE (found version "3.1") found components: C
-- Found PythonInterp: /opt/apps/python/3.10.2/bin/python3.10 (found version "3.10.2")

ADIOS2 build configuration:
  ADIOS Version: 2.8.3
  C++ Compiler : GNU 11.2.0
    /opt/apps/gcc/11.2.0/bin/g++

  Cuda Compiler : /opt/apps/cuda/11.7.1/bin/nvcc
    Cuda Architecture : 52
  Installation prefix: /data/homezvol0/ernestob/sw/hpc3/gpu/adios2-2.8.3
        bin: bin
        lib: lib64
    include: include
      cmake: lib64/cmake/adios2
     python: lib/python3.10/site-packages

  Features:
    Library Type: shared
    Build Type:   Release
    Testing: ON
    Examples: ON
    Build Options:
      BP5       : ON
      DataMan   : OFF
      DataSpaces: OFF
      HDF5      : ON
      HDF5_VOL  : ON
      MHS       : ON
      SST       : ON
      CUDA      : ON
      Fortran   : OFF
      MPI       : ON
      Python    : OFF
      Blosc     : ON
      BZip2     : ON
      LIBPRESSIO: OFF
      MGARD     : OFF
      PNG       : ON
      SZ        : OFF
      ZFP       : OFF
      DAOS      : OFF
      IME       : OFF
      O_DIRECT  : ON
      SysVShMem : ON
      ZeroMQ    : OFF
      Profiling : ON
      Endian_Reverse: OFF
      Sodium    : OFF
    RDMA Transport for Staging: Unconfigured
-- Configuring done
-- Generating done
-- Build files have been written to: /data/homezvol0/ernestob/src/adios2-pm-gpu-build
[  0%] Generating dill_x86_64.c
[  0%] Generating dill.h, dill.c
[  1%] [FLEX][CODScanner] Building scanner with flex 2.6.1
[  1%] Generating dill_virtual.c
[  1%] Building C object thirdparty/perfstubs/perfstubs/CMakeFiles/perfstubs.dir/perfstubs_api/timer.c.o
[  2%] Building C object thirdparty/atl/atl/CMakeFiles/atl.dir/attr.c.o
[  2%] Building C object thirdparty/atl/atl/CMakeFiles/atl.dir/atom.c.o
[  2%] Building C object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/EncodingC.c.o
[  3%] Building CXX object source/adios2/CMakeFiles/adios2_hdf5_mpi.dir/toolkit/interop/hdf5/HDF5CommonMPI.cpp.o
[  3%] Building CXX object source/adios2/CMakeFiles/adios2_hdf5.dir/core/IOHDF5.cpp.o
[  3%] Building CXX object thirdparty/GTest/googletest/googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o
[  3%] Building CUDA object source/adios2/CMakeFiles/adios2_core_cuda.dir/helper/adiosCUDA.cu.o
[  3%] Building C object thirdparty/enet/enet/CMakeFiles/enet.dir/callbacks.c.o
[  4%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_ph5.dir/main.cpp.o
[  4%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_hdf5.dir/main.cpp.o
[  4%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_ascii.dir/main.cpp.o
[  4%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_ascii.dir/HeatTransfer.cpp.o
[  4%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_hdf5.dir/HeatTransfer.cpp.o
[  4%] Generating cod_node.c
Done
[  4%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/dill_util.c.o
[  4%] Built target docs
[  4%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/dill_pkg.c.o
[  5%] Building CXX object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/Directory.cxx.o
[  6%] Building C object thirdparty/enet/enet/CMakeFiles/enet.dir/compress.c.o
[  6%] Building C object thirdparty/enet/enet/CMakeFiles/enet.dir/host.c.o
[  6%] Building C object thirdparty/enet/enet/CMakeFiles/enet.dir/list.c.o
[  6%] Linking C shared library ../../../lib64/libadios2_perfstubs.so
[  6%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_ascii.dir/Settings.cpp.o
[  6%] Built target perfstubs
[  6%] Building CXX object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/DynamicLoader.cxx.o
[  6%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/dill_cplus.c.o
[  6%] Building C object thirdparty/enet/enet/CMakeFiles/enet.dir/packet.c.o
[  6%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/dill_virtual.c.o
[  6%] Building C object thirdparty/enet/enet/CMakeFiles/enet.dir/peer.c.o
[  6%] Building C object thirdparty/enet/enet/CMakeFiles/enet.dir/protocol.c.o
[  6%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_hdf5.dir/Settings.cpp.o
[  6%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_ascii.dir/IO_ascii.cpp.o
[  6%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/virtual.c.o
[  6%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_ph5.dir/HeatTransfer.cpp.o
[  6%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/dill.c.o
[  6%] Building CXX object source/adios2/CMakeFiles/adios2_hdf5.dir/engine/hdf5/HDF5ReaderP.cpp.o
[  6%] Building CXX object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/EncodingCXX.cxx.o
[  6%] Building CXX object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/RegularExpression.cxx.o
[  6%] Building C object thirdparty/atl/atl/CMakeFiles/atl.dir/lookup3.c.o
[  6%] Building CXX object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/SystemTools.cxx.o
[  6%] Building CXX object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/CommandLineArguments.cxx.o
[  6%] Building C object thirdparty/atl/atl/CMakeFiles/atl.dir/tclHash.c.o
[  6%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_hdf5.dir/IO_hdf5_a.cpp.o
[  6%] Linking C shared library ../../../lib64/libadios2_atl.so
[  6%] Built target adios2_hdf5_mpi
[  6%] Building C object thirdparty/enet/enet/CMakeFiles/enet.dir/unix.c.o
[  7%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/vm.c.o
[  7%] Built target atl
[  7%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/dill_x86_64.c.o
[  7%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/x86_64.c.o
[  7%] Building C object thirdparty/dill/dill/CMakeFiles/dill.dir/x86_64_rt.c.o
[  7%] Linking C shared library ../../../lib64/libadios2_enet.so
[  7%] Linking CXX executable ../../../bin/heatTransfer_write_ascii
[  7%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmselect.dir/cmselect.c.o
[  7%] Built target enet
[  8%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_ph5.dir/Settings.cpp.o
[  8%] Building CXX object examples/heatTransfer/write/CMakeFiles/heatTransfer_write_ph5.dir/IO_ph5.cpp.o
[  9%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmsockets.dir/cmsockets.c.o
[  9%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmsockets.dir/ip_config.c.o
[  9%] Built target heatTransfer_write_ascii
[  9%] Building CXX object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/FStream.cxx.o
[  9%] Building CXX object thirdparty/KWSys/adios2sys/CMakeFiles/adios2sys_objects.dir/Status.cxx.o
[  9%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmudp.dir/cmudp.c.o
[  9%] Linking C shared module lib64/libadios2_cmselect.so
[  9%] Linking CXX executable ../../../bin/heatTransfer_write_hdf5
[  9%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmmulticast.dir/cmmulticast.c.o
[  9%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmepoll.dir/cmepoll.c.o
[  9%] Built target cmselect
[  9%] Building CXX object source/adios2/CMakeFiles/adios2_hdf5.dir/engine/hdf5/HDF5WriterP.cpp.o
[  9%] Linking C shared module lib64/libadios2_cmsockets.so
[  9%] Built target cmsockets
[  9%] Building CXX object source/adios2/CMakeFiles/adios2_hdf5.dir/toolkit/interop/hdf5/HDF5Common.cpp.o
[  9%] Built target heatTransfer_write_hdf5
[  9%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmenet.dir/cmenet.c.o
[  9%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmzplenet.dir/cmzplenet.c.o
[  9%] Linking C shared module lib64/libadios2_cmmulticast.so
[  9%] Linking C shared module lib64/libadios2_cmudp.so
[  9%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmzplenet.dir/ip_config.c.o
[  9%] Built target cmmulticast
[  9%] Built target cmudp
[ 10%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/cmenet.dir/ip_config.c.o
[ 10%] Linking C shared module lib64/libadios2_cmepoll.so
[ 10%] Built target cmepoll
[ 10%] Linking C shared module lib64/libadios2_cmenet.so
[ 10%] Linking CXX executable ../../../bin/heatTransfer_write_ph5
[ 10%] Built target cmenet
[ 10%] Built target heatTransfer_write_ph5
[ 10%] Linking C shared library ../../../lib64/libadios2_dill.so
[ 10%] Built target dill
[ 10%] [BISON][CODParser] Building parser with bison 3.0.4
[ 10%] Linking C shared module lib64/libadios2_cmzplenet.so
[ 10%] Built target cmzplenet
[ 10%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/ffs/ffs.c.o
[ 10%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/ffs/ffs_conv.c.o
[ 10%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/ffs/ffs_gen.c.o
[ 10%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/ffs/ffs_formats.c.o
[ 10%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/ffs/ffs_file.c.o
[ 10%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/ffs/unix_io.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/ffs/ffs_marshal.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/ffs/evol.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/version.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/fm/fm_formats.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/fm/fm_dump.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/fm/lookup3.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/fm/string_conversion.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/fm/fm_get.c.o
[ 11%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/fm/xml.c.o
[ 12%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/fm/server_acts.c.o
[ 12%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/fm/unix_io.c.o
[ 12%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/cod/cg.c.o
[ 12%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/cod/standard.c.o
[ 12%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/cod.tab.c.o
[ 12%] Building C object thirdparty/ffs/ffs/CMakeFiles/ffs.dir/cod_node.c.o
[ 12%] Built target adios2sys_objects
[ 12%] Linking C shared library ../../../lib64/libadios2_ffs.so
[ 13%] Built target ffs
[ 13%] Generating cm_interface.c, revp.c, revpath.h
done
[ 13%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm.c.o
[ 13%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_control.c.o
[ 13%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_formats.c.o
[ 13%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_util.c.o
[ 13%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_transport.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_perf.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_lock.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_pbio.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_interface.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/version.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_threadio.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/cm_evol.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/evp.c.o
[ 14%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/response.c.o
[ 15%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/metrics.c.o
[ 15%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/dlloader.c.o
[ 15%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/ip_config.c.o
[ 15%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/chr_time.c.o
[ 15%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/revp.c.o
[ 15%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/evp_compat.c.o
[ 15%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/thin_server.c.o
[ 15%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/evp_threads.c.o
[ 16%] Building C object thirdparty/EVPath/EVPath/CMakeFiles/EVPath.dir/ev_dfg.c.o
[ 16%] Linking C shared library ../../../lib64/libadios2_evpath.so
[ 16%] Built target EVPath
[ 16%] Building C object source/adios2/toolkit/sst/CMakeFiles/sst.dir/dp/dp.c.o
[ 16%] Building C object source/adios2/toolkit/sst/CMakeFiles/sst.dir/dp/dummy_dp.c.o
[ 16%] Building C object source/adios2/toolkit/sst/CMakeFiles/sst.dir/dp/evpath_dp.c.o
[ 16%] Building C object source/adios2/toolkit/sst/CMakeFiles/sst.dir/cp/cp_reader.c.o
[ 16%] Building C object source/adios2/toolkit/sst/CMakeFiles/sst.dir/cp/cp_writer.c.o
[ 16%] Building C object source/adios2/toolkit/sst/CMakeFiles/sst.dir/cp/cp_common.c.o
[ 17%] Building CXX object source/adios2/toolkit/sst/CMakeFiles/sst.dir/sst_comm.cpp.o
[ 17%] Building C object source/adios2/toolkit/sst/CMakeFiles/sst.dir/cp/ffs_marshal.c.o
[ 17%] Built target sst
[ 17%] Built target adios2_hdf5
[ 17%] Linking CXX static library ../../../../lib/libgtest.a
[ 17%] Built target gtest
[ 17%] Linking CUDA device code CMakeFiles/adios2_core_cuda.dir/cmake_device_link.o
[ 17%] Linking CUDA shared library ../../lib64/libadios2_core_cuda.so
[ 17%] Built target adios2_core_cuda
[ 17%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/common/ADIOSTypes.cpp.o
[ 17%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/Attribute.cpp.o
[ 17%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/AttributeBase.cpp.o
[ 17%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/ADIOS.cpp.o
[ 17%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/Engine.cpp.o
[ 17%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/IO.cpp.o
[ 18%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/Operator.cpp.o
[ 18%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/Variable.cpp.o
[ 18%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/Stream.cpp.o
[ 18%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/VariableBase.cpp.o
[ 18%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/VariableCompound.cpp.o
[ 18%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/core/Group.cpp.o
[ 18%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/operator/callback/Signature1.cpp.o
[ 18%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/operator/callback/Signature2.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/operator/compress/CompressNull.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/operator/OperatorFactory.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosComm.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosCommDummy.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosDynamicBinder.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosMath.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosMemory.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosNetwork.cpp.o
[ 19%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosPluginManager.cpp.o
[ 20%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosSystem.cpp.o
[ 20%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosString.cpp.o
[ 20%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosType.cpp.o
[ 20%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosXML.cpp.o
[ 20%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosXMLUtil.cpp.o
[ 20%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosYAML.cpp.o
[ 20%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosLog.cpp.o
[ 20%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/helper/adiosRangeFilter.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/bp3/BP3Reader.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/bp3/BP3Writer.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/bp4/BP4Reader.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/bp4/BP4Writer.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/skeleton/SkeletonReader.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/skeleton/SkeletonWriter.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/inline/InlineReader.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/inline/InlineWriter.cpp.o
[ 21%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/null/NullWriter.cpp.o
[ 22%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/null/NullReader.cpp.o
[ 22%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/plugin/PluginEngine.cpp.o
[ 22%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/engine/plugin/PluginEngineInterface.cpp.o
[ 22%] Building CXX object source/adios2/CMakeFiles/adios2_core.dir/operator/plugin/PluginOperator.cpp.o
g++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
gmake[2]: *** [source/adios2/CMakeFiles/adios2_core.dir/build.make:146: source/adios2/CMakeFiles/adios2_core.dir/core/IO.cpp.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
gmake[1]: *** [CMakeFiles/Makefile2:3637: source/adios2/CMakeFiles/adios2_core.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
[ernestob@login-i15:~] $

```

--- a/Docs/source/install/hpc/hpc3.rst
+++ b/Docs/source/install/hpc/hpc3.rst
@@ -96,8 +96,8 @@ Use the following :ref:`cmake commands <building-cmake>` to compile:
    rm -rf build
 
    cmake -S . -B build -DWarpX_COMPUTE=CUDA -DWarpX_PSATD=ON -DWarpX_QED_TABLE_GEN=ON -DWarpX_LIB=ON -DWarpX_DIMS="1;2;RZ;3"
-   cmake --build build -j 16
-   cmake --build build -j 16 --target pip_install
+   cmake --build build -j 12
+   cmake --build build -j 12 --target pip_install
 
 **That's it!**
 The WarpX application executables are now in ``$HOME/src/warpx/build/bin/`` and we installed the ``pywarpx`` Python module.

--- a/Tools/machines/hpc3-uci/install_gpu_dependencies.sh
+++ b/Tools/machines/hpc3-uci/install_gpu_dependencies.sh
@@ -56,7 +56,7 @@ else
 fi
 rm -rf $HOME/src/c-blosc-pm-gpu-build
 cmake -S $HOME/src/c-blosc -B $HOME/src/c-blosc-pm-gpu-build -DBUILD_TESTS=OFF -DBUILD_BENCHMARKS=OFF -DDEACTIVATE_AVX2=OFF -DCMAKE_INSTALL_PREFIX=${SW_DIR}/c-blosc-1.21.1
-cmake --build $HOME/src/c-blosc-pm-gpu-build --target install --parallel 16
+cmake --build $HOME/src/c-blosc-pm-gpu-build --target install --parallel 12
 rm -rf $HOME/src/c-blosc-pm-gpu-build
 
 # ADIOS2
@@ -71,7 +71,7 @@ else
 fi
 rm -rf $HOME/src/adios2-pm-gpu-build
 cmake -S $HOME/src/adios2 -B $HOME/src/adios2-pm-gpu-build -DADIOS2_USE_Blosc=ON -DADIOS2_USE_Fortran=OFF -DADIOS2_USE_Python=OFF -DADIOS2_USE_ZeroMQ=OFF -DCMAKE_INSTALL_PREFIX=${SW_DIR}/adios2-2.8.3
-cmake --build $HOME/src/adios2-pm-gpu-build --target install -j 16
+cmake --build $HOME/src/adios2-pm-gpu-build --target install --parallel 12
 rm -rf $HOME/src/adios2-pm-gpu-build
 
 # BLAS++ (for PSATD+RZ)
@@ -87,7 +87,7 @@ else
 fi
 rm -rf $HOME/src/blaspp-pm-gpu-build
 cmake -S $HOME/src/blaspp -B $HOME/src/blaspp-pm-gpu-build -Duse_openmp=OFF -Dgpu_backend=cuda -DCMAKE_CXX_STANDARD=17 -DCMAKE_INSTALL_PREFIX=${SW_DIR}/blaspp-master
-cmake --build $HOME/src/blaspp-pm-gpu-build --target install --parallel 16
+cmake --build $HOME/src/blaspp-pm-gpu-build --target install --parallel 12
 rm -rf $HOME/src/blaspp-pm-gpu-build
 
 # LAPACK++ (for PSATD+RZ)
@@ -103,7 +103,7 @@ else
 fi
 rm -rf $HOME/src/lapackpp-pm-gpu-build
 CXXFLAGS="-DLAPACK_FORTRAN_ADD_" cmake -S $HOME/src/lapackpp -B $HOME/src/lapackpp-pm-gpu-build -DCMAKE_CXX_STANDARD=17 -Dbuild_tests=OFF -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON -DCMAKE_INSTALL_PREFIX=${SW_DIR}/lapackpp-master
-cmake --build $HOME/src/lapackpp-pm-gpu-build --target install --parallel 16
+cmake --build $HOME/src/lapackpp-pm-gpu-build --target install --parallel 12
 rm -rf $HOME/src/lapackpp-pm-gpu-build
 
 

